llama_index/node_parser/simple.py

```diff
@@ -42,6 +48,9 @@ def get_nodes_from_documents(
             include_extra_info (bool): whether to include extra info in nodes
 
         """
+        event_id = self._callback_manager.on_event_start(
+            CBEventType.NODE_PARSING, payload={"documents": documents}
```

Comment:
curious: how do you imagine handlers to handle arbitrary objects in the payload?

Reply:
I see most of these payloads as just a "visual" insight into what's happening for certain events.

So handlers could use these payloads and display them nicely, if they want. 

On top of the payloads, you also have time and occurrence information to pull as well.